resources/telemetry/charts/fluent-bit/values.yaml

```diff
@@ -236,21 +246,33 @@ envFrom:
       name: telemetry-fluent-bit-env
       optional: true
 
-extraContainers: []
-#   - name: do-something
-#     image: busybox
-#     command: ['do', 'something']
+extraContainers: |
+  - name: exporter
+    image: {{ include "imageurl" (dict "reg" .Values.global.containerRegistry "img" .Values.global.images.directory_size_exporter) }}
+    resources:
+      limits:
+        cpu: 100m
+        memory: 50Mi
+      requests:
+        memory: 5Mi
+    ports:
+      - name: http-metrics
+        containerPort: 2021
+        protocol: TCP
+    volumeMounts:
+      - name: varfluentbit
+        mountPath: /data
 
 flush: 1
 
 metricsPort: 2020
 
-extraPorts: []
-#   - port: 5170
-#     containerPort: 5170
-#     protocol: TCP
-#     name: tcp
-#     nodePort: 30517
+extraPorts:
+  - name: http-metrics
```

Comment:
Let's give the port a more meaningful name. For instance, `http-storage-metrics`. Same in the ServiceMonitor.